app/repositories/IndexesManager.scala

```diff
@@ -80,6 +80,9 @@ trait IndexesManager extends Logging {
       wildcardProjection = None
     )
 
+    logger.info(s"[IndexesManager]: Collection name: $collectionName")
+    logger.info(s"[IndexesManager]: idIndex: $idIndex")
```

Comment:
This will log the idIndex as we have it defined in our app. We want to log the indexes as they appear in Mongo. I've added a comment above to show how this can be done.